audb/core/api.py

```diff
@@ -26,12 +26,19 @@ def available(
     r"""List all databases that are available to the user.
 
     Args:
-        only_latest: keep only latest version
+        only_latest: include only latest version of database
 
     Returns:
-        table with name, version and private flag
+        table with database name as index,
+        and backend, host, repository, version as columns
 
-    """
+    Example:
+        >>> audb.available(only_latest=True)
```

Comment:
If we add another database it will break the example, so maybe it's safer to add a `# pragma: no cover` here?

Reply:
The fear of adding another database was the reason why didn't had this example before. But I'm not so sure at the moment if we will add another database. Even if we go for further public databases, we might publish them on a server shared with university. Or we have to change the example, which still feels safer than excluding it from the tests.